### PR TITLE
Remove menu border bottom

### PR DESCRIPTION
**Pull request by ClaudivanFilho**
#### What is the purpose of this pull request?
- Remove border-bottom of menu.

#### How should this be manually tested?
https://claudivan--storecomponents.myvtex.com/make-b--brilho-labial-star-lip-7ml-166starlipbril/p

### Desk:top:
screenshot from 2018-08-14 15-27-34
### Tablet
screenshot from 2018-08-14 15-29-10
### Mobile
screenshot from 2018-08-14 15-28-23

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- remove border-bottom of menu.
```
Comment (vitoriaHeliane) [inline CHANGELOG.md]: Coloca menu entre aspas.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- remove border-bottom of `menu`.
```
Comment (salesfelipe) [inline CHANGELOG.md]: from

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- remove border-bottom from `menu`.
+- height of `menu` to 1.5625rem.
```
Comment (lucasecdb) [inline CHANGELOG.md]: I don't think we need to put this level of detail on the changelog, something like "decreased menu height" would be better.

Comment (andreldsa) [reply CHANGELOG.md]: Agreed

**Pull request merged**